lib/Minz/View.php

```diff
@@ -237,6 +238,37 @@ public static function appendStyle($url, $media = 'all', $cond = false) {
 		);
 	}
 
+	/**
+	 * @param array $themeColors
+	 * @return void
+	 */
+	public static function appendThemeColors($themeColors): void {
+		self::$themeColors = $themeColors;
+	}
+
+	/**
+	 * https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name/theme-color
+	 *
+	 */
+	public static function metaThemeColor(): String {
+		$meta = '';
+
+		if (!empty(self::$themeColors['light'])) {
+			$meta .= '<meta name="theme-color" media="(prefers-color-scheme: light)" content="' . self::$themeColors['light'] . '" />';
+		}
+		if (!empty(self::$themeColors['dark'])) {
+			$meta .= '<meta name="theme-color" media="(prefers-color-scheme: dark)" content="' . self::$themeColors['dark'] . '" />';
+		}
+		if (!empty(self::$themeColors['default'])) {
+			$meta .= '<meta name="theme-color" content="' . self::$themeColors['default'] . '" />';
+		}
+		if (empty(self::$themeColors['default']) && !empty(self::$themeColors && empty(self::$themeColors['light']) && empty(self::$themeColors['dark']))) {
```

Comment:
```suggestion
		if (empty(self::$themeColors['default']) && !empty(self::$themeColors) && empty(self::$themeColors['light']) && empty(self::$themeColors['dark'])) {
```

Reply:
You can just click on *Commit suggestion* from the Web UI (or from e.g. the GitHub extension in VSCode) in order to avoid the *Outdated* status and the need to manually resolve conversation. Another benefit is to preserve credits to the suggestion's author

Reply:
I know it and I did it :)

Reply:
Ah, sorry :-)